R/utils_asserts.R

```diff
@@ -1,64 +1,88 @@
-#' Assert that columns exist in a data frame
+# Unclassified ------------------------------------------------------------
```

Comment:
Please don't remove existing assertions for now. I am improperly importing within rapporteur at the moment and this should be part of the wider who says package conversation.